Cargo.toml

```diff
@@ -33,13 +33,11 @@ regex = "~0.1.69"
 default     = ["suggestions", "color", "wrap_help"]
 suggestions = ["strsim"]
 color       = ["ansi_term", "libc"]
-yaml        = ["yaml-rust"]
 wrap_help   = ["libc", "term_size"]
-lints       = ["clippy", "nightly"]
-nightly     = [] # for building with nightly and unstable features
```

Comment:
I think we should keep `nightly` in order to differentiate between "clap unstable features (can sometimes build on stable Rust)" and "Rust unstable features, requires nightly compiler"